js/login.js

```diff
@@ -1,15 +1,15 @@
 function setNameIfFound(name){
-    const userLogin = document.querySelector("#login-btn")
+    const userLogin = document.querySelector("#login-btn");
     if(name){
-        userLogin.innerHTML = `Hello ${name}!`
-    }
-}
+        userLogin.innerHTML = `Hello ${name}!`;
+    };
+};
 
-let url = "https://staging-api.realdevsquad.com/users/self"
+let url = "https://staging-api.realdevsquad.com/users/self";
 
 fetch(url, {
```

Comment:
What if we could call the fetch after the DOM Content is loaded?
We can directly shift this code inside the eventListener of DOMContentLoaded but it wouldn't be much readable. For that we could wrap the fetch call in a function and pass that function to the eventListener.

Why am I saying so is that if someone moves the `<script` tag in the `<head>` , since the JS has referred the some DOM content, it won't find it and fail.

Read more here: https://stackoverflow.com/questions/34807614/should-i-wait-for-domcontentloaded-event-if-i-place-a-script-tag-at-the-end-of-a#:~:text=If%20script%20modifies%20element%2C%20it,until%20it's%20sure%20it%20exists.

Reply:
Done with the changes. Thanks for the review!